spec/models/user_spec.rb

```diff
@@ -345,24 +345,35 @@
     end
 
     describe 'proposals methods' do
-      let(:submitter) { create(:submitter, user: user) }
+      let(:submitter) { create(:user) }
+      let(:speaker) { create(:user) }
       let(:event1) { create(:event, program: conference.program) }
       let(:event2) { create(:event, program: conference.program) }
+      let(:event3) { create(:event, program: conference.program) }
```

Comment:
Why do you need a third event for that, use one of the others. The case when we have both a submitter and an speaker is the one we want to test. 😉

Reply:
Yes you are right i will correct it!